src/main/java/com/google/sps/MutationTest.java

```diff
@@ -74,7 +74,10 @@ public void addNodeBasic() {
     Mutation addA = Mutation.newBuilder().setType(Mutation.Type.ADD_NODE).setStartNode("A").build();
 
     HashSet<String> roots = new HashSet<>();
-    boolean success = Utility.mutateGraph(addA, graph, graphNodesMap, roots);
+
```

Comment:
nit: make these newlines consistent with other tests? The same applies to the other tests as well. For example, roots and dataGraph could always be a single block while mutateGraph and assert can be another block.